scripts/plot/make_plots.py

```diff
@@ -77,16 +79,26 @@
 
 selection = {
     'trigger'  : (r'Trigger'),
-    'baseline' : (r'Trigger'+'\n'+
-                  r'Dilepton cuts')
+    'dilepton_SR' : (r'Trigger'+'\n'+
```

Comment:
This is hardcoded in the plotting code.. we will need to remove it in a next PR.